CONTRIBUTING.md

```diff
@@ -4,7 +4,9 @@ Welcome to the Nipype repository! We're excited you're here and want to contribu
 
 These guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
 
-Before you start you'll need to set up a free [GitHub][link_github] account and sign in. Here are some [instructions][link_signupinstructions].
+Before you start you'll need to set up a free [GitHub][link_github] account and sign in, here are some [instructions][link_signupinstructions].
+If you are not familiar with version control system and Git, 
+you will find introduction and links to tutorials [here](http://www.reproducibleimaging.org/module-reproducible-basics/02-vcs/).
```

Comment:
Linking with "here" as the text can usually be improved by picking a more descriptive word or phrase and centering the text around that. One option:

```Markdown
[introductions and tutorials](<URL>) may be found on [ReproducibleImaging.org](https://www.reproducibleimaging.org/).
```